homeassistant/components/litterrobot/manifest.json

```
@@ -4,6 +4,7 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/litterrobot",
   "requirements": ["pylitterbot==2022.8.0"],
+  "dhcp": [{ "registered_devices": true }, { "hostname": "litter-robot4" }],
```

```suggestion
  "dhcp": [{ "hostname": "litter-robot4" }],
```

Since you don't need to update the ip address, `registered_devices` isn't so useful here.